### DISCUSSION TITLE
不知道是artalk还是模板的问题

### DISCUSSION THREAD

**Discussion by penndu** [volantis-x/hexo-theme-volantis#746] [Q&A] [answered]
通过邮件或者控制中心无法定位评论，如图：

https://bu.dusays.com/2022/04/26/6267fdab40586.gif

**Comment by penndu** [comment]
详细描述一下，通过控制中心跳转评论，会出现空白屏，需向上移动页面才会显示出评论，但评论先之后，没有定位评论。

**Comment by inkss** [comment, marked as answer]
这是由于开启了 scrollreveal 导致的，暂时性的解决方案为：

1.关闭 scrollreveal 功能。

2.去掉评论区域的滚动动画，修改 [comments/index.ejs](https://github.com/volantis-x/hexo-theme-volantis/blob/dev/layout/_plugins/comments/index.ejs#L9) 文件，将 `reveal` 删去。
